test/parallel/test-assert.js

```diff
@@ -406,7 +406,7 @@ assert.doesNotThrow(function() { assert.ifError(); });
 
 assert.throws(() => {
   assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
-}, /Got unwanted exception: user message/,
+}, /Got unwanted exception:\ntest: user message/,
               'a.doesNotThrow ignores user message');
```

Comment:
This message is no longer accurate...this test fails if `a.doesNotThrow` ignores user message **or if it ignores the message of the unwanted exception**..